bindinfo/cache.go

```diff
@@ -34,7 +33,8 @@ const (
 // BindMeta stores the basic bind info and bindSql astNode.
 type BindMeta struct {
 	*BindRecord
-	Ast ast.StmtNode //ast will be used to do query sql bind check
+	// Hint is the parsed hints, it is used to bind hints to stmt node.
+	Hint *HintsSet
```

Comment:
`BindMeta` has been removed already in latest master branch?

Reply:
Yes, master has to support multiply bindings for a sql while 3.0 does not need to.